src/app/app.module.ts

```diff
@@ -1,18 +1,22 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms'; 
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { HomePageComponent } from './home-page/home-page.component';
+import { CreateChatroomModalComponent } from './create-chatroom-modal/create-chatroom-modal.component';
+import { Ng2SearchPipeModule } from 'ng2-search-filter';
 
 @NgModule({
   declarations: [
     AppComponent,
-    HomePageComponent
+    CreateChatroomModalComponent
```

Comment:
(suggestion) It is better to have a separate module for a given feature, that is then imported in the app.module.